src/main/java/com/google/ehub/servlets/DashboardServlet.java

```diff
@@ -0,0 +1,69 @@
+package com.google.ehub.servlets;
+
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.ehub.data.EntertainmentItem;
+import com.google.ehub.data.EntertainmentItemDatastore;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/*
+ * Handles GET requests to retrieve EntertainmentItem entities
+ * stored in Datastore and make them available to the Dashboard.
+ */
+@WebServlet("/dashboard")
+public class DashboardServlet extends HttpServlet {
+  private static final String SEARCH_VALUE_PARAMETER_KEY = "searchValue";
+  private static final String SORTING_DIRECTION_PARAMETER_KEY = "sortingDirection";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String searchValue = request.getParameter(SEARCH_VALUE_PARAMETER_KEY);
+    String sortingDirection = request.getParameter(SORTING_DIRECTION_PARAMETER_KEY);
+
+    if (!areGetRequestParametersValid(searchValue, sortingDirection)) {
+      System.err.println("DashboardServlet: Get Request parameters not specified!");
+      return;
+    }
+
+    SortDirection sortDirEnum = SortDirection.valueOf(sortingDirection);
```

Comment:
tiny nit: usually we don't use "enum" as part of var names
`SortDirection sortDir` is fine